src/gridcoin/researcher.cpp

```diff
@@ -32,6 +32,13 @@ extern CCriticalSection cs_main;
 extern std::string msMiningErrors;
 extern unsigned int nActiveBeforeSB;
 
+const std::vector<MiningPool> MiningPools::GetMiningPools()
```

Comment:
Since this returns a copy you probably want to return a non-const here.

Reply:
ok